test/ANNOTATIONS.yaml

```diff
@@ -1158,6 +1160,10 @@ memleaksfull:
     - Access param, type fields without postfix-! (#14549)
   12/06/19:
     - Close user-level leaks in some new tests (#14586)
+  12/06/19:
```

Comment:
This should be melded into the previous line (repeated dates will override) (see for example the annotations for `03/12/19:`)

Reply:
and I read the comment in the beginning of the file less than 1 minute before doing this..